src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/kafka/KafkaSyketilfellebit.kt

```diff
@@ -15,3 +16,10 @@ data class KafkaSyketilfellebit(
     val tom: LocalDate,
     val korrigererSendtSoknad: String?,
 )
+
+fun KafkaSyketilfellebit.isRelevantForOppfolgingstilfelle(): Boolean = isArbeidstakerBit() || isNotArbeidstakerBit()
```

Comment:
Hvilke biter er egentlig *ikke* relevante?

Reply:
Godt spørsmål. Jeg har ikke full oversikt over alle varianter av SyketilfelleBit. Jeg tror vi lagrer det aller meste her, men ikke nødvendigvis alt. Det kan også forandre seg i fremtiden. Jeg må evt. se litt mer i databasen til Syfosyketilfelle for å undersøke mer.

Reply:
Føles litt forvirrende dette utrykket isArbeidstakerBit() || isNotArbeidstakerBit(). Virker som den egentlig er relevant uavhengig av om det er arbeidstaker eller ei?
Kunne man sagt isArbeidstakerBit() || isBekreftetSykmeldingNotArbeidstakerBit() f.eks,

Reply:
Har sjekket i databasen nå og biter som ikker er relevant er som følger:
Sykmelding-Avbrutt(ikke helt sikker på hva denne er, men ser ut som noe legacy for sykmeldinger som ikke er tatt i bruk) og Sykmelding-Ny(ikke innsendte eller bekreftede sykmeldinger, aka ubrukt).